cpp/src/io/parquet/reader_impl.cpp

```diff
@@ -607,7 +607,8 @@ parquet_metadata read_parquet_metadata(host_span<std::unique_ptr<datasource> con
   return parquet_metadata{parquet_schema{walk_schema(&metadata, 0)},
                           metadata.get_num_rows(),
                           metadata.get_num_row_groups(),
-                          metadata.get_key_value_metadata()[0]};
+                          metadata.get_key_value_metadata()[0],
+                          std::move(metadata.get_rowgroup_metadata())};
```

Comment:
I don't think we need std::move here. The result of `get_rowgroup_metadata` is already an rvalue.
```suggestion
                          metadata.get_rowgroup_metadata()};
```

Reply:
Sounds good. Let me push the update

Reply:
Pushed an update